.lando.base.yml

```diff
@@ -46,38 +53,12 @@ tooling:
   npm:
     service: appserver
     cmd: /usr/bin/npm
-  yarn:
```

Comment:
Pulling out all nightwatch stuff as I know it's rarely used now. Maybe not 100% gone but not used regularly enough to make sure this works end-to-end every time. I'd be happy to pull it out of the git logs from other projects and adapt as/when it might be needed in future, and leave the bulk of automated functional testing to the QA team